diff --git a/tests/ExampleFile.java b/tests/ExampleFile.java
index d340f6a..29b54da 100644
--- a/tests/ExampleFile.java
+++ b/tests/ExampleFile.java
@@ -1,13 +1,5 @@
 package com.mydomain.myapp;

 public class JavaTests {
-    // TODO: Expand this class
-    //  Do it
-    // labels: urgent

-    /*
-    TODO: Definitely expand this class
-    Do it ASAP
-    labels: urgent
-    */
 }
\ No newline at end of file
diff --git a/tests/example-file.php b/tests/example-file.php
deleted file mode 100644
index 063bb80..0000000
--- a/tests/example-file.php
+++ /dev/null
@@ -1,23 +0,0 @@
-<!DOCTYPE html>
-<html>
-<body>
-<!-- TODO: Check HTML comments work -->
-<?php
-    echo 'Testing 123';
-    // TODO: Expand this page
-    //  We aren't doing anything here yet
-
-    echo 'Testing 456';
-    # TODO: Definitely expand this
-    #  There needs to be some point to this file
-
-    echo 'Testing 789'
-    /*
-     * TODO: Echo a greeting instead
-     * This should work with or without the asterisk on each line
-     * labels: help wanted
-     */
-?>
-
-</body>
-</html>
\ No newline at end of file
diff --git a/tests/example_file.py b/tests/example_file.py
index 525e25d..ba4e68d 100644
--- a/tests/example_file.py
+++ b/tests/example_file.py
@@ -1,23 +1,2 @@
 def hello_world():
-    # TODO: Come up with a more imaginative greeting
     print('Hello world')
-
-    # TODO: Do more stuff
-    #  This function should probably do something more interesting
-    #  labels: help wanted
-
-def my_method(arg):
-    """
-    TODO: Second multiline comment
-    also need to be turned into task, and hopefully
-    kept together as one.
-    """
-    pass
-
-def MyClass:
-    '''
-    TODO: Third multiline comment
-    also need to be turned into task, and hopefully
-    kept together as one.
-    '''
-    pass
diff --git a/tests/example_file.cr b/tests/example_file.cr
index e6da2ec..67f14dd 100644
--- a/tests/example_file.cr
+++ b/tests/example_file.cr
@@ -1,14 +1,3 @@
-# TODO: Come up with a more imaginative greeting
 puts "Greetings"
-
-# TODO: Do even more stuff
-# This function should probably do something more interesting
-# labels: help wanted
diff --git a/tests/example_file.rb b/tests/example_file.rb
index e6da2ec..67f14dd 100644
--- a/tests/example_file.rb
+++ b/tests/example_file.rb
@@ -1,14 +1,3 @@
 #!/usr/bin/ruby -w

-# TODO: Come up with a more imaginative greeting
 puts "Greetings"
-
-=begin
-TODO: Do more stuff
-This function should probably do something more interesting
-labels: help wanted
-=end
-
-# TODO: Do even more stuff
-# This function should probably do something more interesting
-# labels: help wanted
diff --git a/tests/example_file.yaml b/tests/example_file.yaml
index 6397789..494d42f 100644
--- a/tests/example_file.yaml
+++ b/tests/example_file.yaml
@@ -1,7 +1,3 @@
 name: "TODO to Issue"
-# TODO: Write a more interesting description
 description: "Converts IDE TODO comments to GitHub issues"
 author: "Alastair Mooney"
-# TODO: Define inputs
-#  Need to do this before the action is released
-#  labels: urgent
diff --git a/tests/example_file.prog.abap b/tests/example_file.prog.abap
index 6397789..494d42f 100644
--- /dev/null
+++ b/tests/example_file.prog.abap
@@ -1,5 +0,0 @@
-REPORT ztest_todo_2.
-
-DATA moo TYPE i VALUE 2.
-WRITE moo. " TODO This is an end-of-line to-do
-moo = 4. * TODO This is another end-of-line to-do
diff --git a/tests/example_file.sql b/src/tests/example_file.sql
index 6397789..494d42f 100644
--- /dev/null
+++ b/src/tests/example_file.sql
@@ -1,2 +0,0 @@
--- TODO Select all:
-SELECT * FROM Products;
diff --git a/tests/example_file.tex b/src/tests/example_file.tex
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.tex
@@ -0,0 +1,2 @@
-% TODO Add in preamble details
-\begin{document}
-  \begin{comment}
-     TODO This document needs content
-     label: urgent
-  \end{comment}
-\end{document}
diff --git a/tests/example_file.jl b/src/tests/example_file.jl
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.jl
@@ -0,0 +1,2 @@
- #TODO: Hopefully this comment turns into an issue
- # TODO: Hopefully this comment also turns into an issue
- print("Hello World")
- #= TODO: Multiline comments
-   also need to be turned into task, and hopefully
-   kept together as one.
- =#
- #=TODO: Another copied multiline comment
-   also need to be turned into task, and hopefully
-   kept together as one.
- =#
diff --git a/tests/defs.bzl b/tests/defs.bzl
index 525e25d..ba4e68d 100644
--- a/tests/defs.bzl
+++ b/tests/defs.bzl
@@ -0,0 +0,1 @@
-    # TODO: Come up with a more imaginative greeting
diff --git a/tests/example_file.ahk b/src/tests/example_file.ahk
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.ahk
@@ -0,0 +1,2 @@
- ; TODO: Find a better way to manage hotkeys
- ; Maybe just switch to vim??
- #h::
- RegRead, HiddenFiles_Status, HKEY_CURRENT_USER, Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced, Hidden
diff --git a/tests/example_file.hbs b/src/tests/example_file.hbs
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.hbs
@@ -0,0 +1,2 @@
- <!-- TODO: Hopefully this comment turns into a todo issue -->
- {{!
-   TODO: Make a handlebar templtate
-    This is really just a test, but hopefully this works~!
- }}
diff --git a/tests/example_file.org b/src/tests/example_file.org
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.org
@@ -0,0 +1,2 @@
- # TODO: Hopefully this comment turns into a todo issue
- #+begin_src python
- print("Hello World")
- #+end_src
- #+begin_comment
-  TODO: Multiline comments
-   also need to be turned into todos, and hopefully
-   kept together as one todo
- #+end_comment
diff --git a/tests/example_file.scss b/tests/example_file.scss
index 525e25d..ba4e68d 100644
--- a/tests/example_file.scss
+++ b/tests/example_file.scss
@@ -1,11 +0,0 @@
-// TODO: Hopefully this comment turns into a todo issue
-.my-class {
-  font-weight: bold;
-  .my-second-class {
-    /* TODO: Multiline comments
-     * also need to be turned into todos, and hopefully
-     * kept together as one todo
-     */
-    text-align: center;
-  }
-}
diff --git a/tests/example_file.twig b/src/tests/example_file.twig
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.twig
+++ b/src/tests/example_file.twig
@@ -1,6 +0,0 @@
- {# TODO: Hopefully this comment turns into a todo issue #}
- {#
-   TODO:Multiline comments
-   also need to be turned into todos, and hopefully
-   kept together as one todo
- #}
diff --git a/tests/example_file.md b/src/tests/example_file.md
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.md
+++ b/src/tests/example_file.md
@@ -1,7 +0,0 @@
- {/* TODO: Hopefully this comment turns into a todo issue */}
- {/*
-   TODO:Multiline comments
-   also need to be turned into todos, and hopefully
-   kept together as one todo
- */}
- - [ ] TODO: An inline todo that's NOT a comment (what)
diff --git a/tests/example_file.mdx b/src/tests/example_file.mdx
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.mdx
+++ b/src/tests/example_file.mdx
@@ -1,7 +0,0 @@
- {/* TODO: Hopefully this comment turns into a todo issue */}
- {/*
-   TODO:Multiline comments
-   also need to be turned into todos, and hopefully
-   kept together as one todo
- */}
- - [ ] TODO: An inline todo that's NOT a comment (what)
diff --git a/tests/example_file.r b/tests/example_file.r
index 525e25d..ba4e68d 100644
--- a/tests/example_file.r
+++ b/tests/example_file.r
@@ -1,23 +1,2 @@
hello_world <- function() {
-    # TODO: Come up with a more imaginative greeting
     message("Hello world")
-
-    # TODO: Do more stuff
-    #  This function should probably do something more interesting
-    #  labels: help wanted
}
diff --git a/tests/example_file.rmd b/src/tests/example_file.rmd
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.rmd
+++ b/src/tests/example_file.rmd
@@ -0,1 +0,0 @@
- <!-- TODO: Check RMarkdown comments work -->
diff --git a/tests/example_file.qmd b/src/tests/example_file.qmd
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.qmd
+++ b/src/tests/example_file.qmd
@@ -0,1 +0,0 @@
- <!-- TODO: Check RMarkdown comments work -->
diff --git a/src/Swarm/Game/example.hs b/src/Swarm/Game/example.hs
index 525e25d..ba4e68d 100644
--- a/src/Swarm/Game/example.hs
+++ b/src/Swarm/Game/example.hs
@@ -1,14 +0,0 @@
--- | Standard devices that are always installed.
---
--- TODO: Remove standard devices
---  In the future, make a way to build these and just start the base
---  out with a large supply of each?
---  labels: redesign
-stdDeviceList =
-  ["treads", "grabber", "solar panel", "scanner", "plasma cutter"]
-
-{- | Very complicated function.
-
-TODO: Create an issue for TODO
--}
-sum a b = a + b
diff --git a/tests/config.jsonc b/tests/config.jsonc
index 0000000..525e25d 100644
--- a/tests/config.jsonc
+++ b/tests/config.jsonc
@@ -0,0 +0,1 @@
- // TODO: Delete this line from the codebase
diff --git a/tests/config.json5 b/tests/config.json5
index 0000000..525e25d 100644
--- a/tests/config.json5
+++ b/tests/config.json5
@@ -0,0 +0,1 @@
- // TODO: Delete this line from the codebase
diff --git a/tests/example.clj b/tests/example.clj
index 0000000..525e25d 100644
--- a/tests/example.clj
+++ b/tests/example.clj
@@ -1,8 +0,0 @@
-(ns example)
-
-(defn function-test [name]
-  ;; TODO: first todo to test parser
-  (prn (str "hello" name))
-  ;; TODO: todo with description (sub-level)
-  ;;  Body of the issue should stay here
-  (str "hello" name))
diff --git a/tests/example.gd b/tests/example.gd
index 0000000..525e25d 100644
--- a/tests/example.gd
+++ b/tests/example.gd
@@ -1,5 +0,0 @@
-# TODO Store all the players here
-var _players: = []
-
-# TODO Play the level music
-export: var music_path: String
diff --git a/tests/example.nix b/tests/example.nix
index a6c6cb0..25d195f 100644
--- a/tests/example.nix
+++ b/tests/example.nix
@@ -1,11 +1,7 @@
 {
-  # TODO add missing devices
   stdDeviceList = [
     "treads"
     "grabber"
   ];
-  /*
-    TODO add missing c
-  */
   sum = a: b: a + b;
 }
diff --git a/tests/example.xaml b/tests/example.xaml
new file mode 100644
index 0000000..a6c6cb0
--- /dev/null
+++ b/tests/example.xaml
@@ -1,4 +1,2 @@
-<!-- TODO: Add xaml support -->
-<Label Text="why"/>
-<!-- TODO: please -->
-<Label Text="ok"/>
<Label Text="There you go"/>
<Label Text="Thank you"/>
diff --git a/tests/example.bas b/tests/example.bas
index 6b0c6cf..b37e70a 100644
--- a/tests/example.bas
+++ b/tests/example.bas
@@ -1,2 +1,1 @@
-' TODO: simplify
-Public Sub MakeThingsComplicated()
+Public Sub DoItSimpler()
diff --git a/tests/example.frm b/tests/example.frm
index 6b0c6cf..b37e70a 100644
--- a/tests/example.frm
+++ b/tests/example.frm
@@ -1,2 +1,1 @@
-' TODO: remove feature to prevent legal issues
-Public Sub Plagiarize()
+Public Sub MakeOriginalIdea()
diff --git a/tests/example.cu b/tests/example.cu
index 0000000..a6c6cb0
--- /dev/null
+++ b/tests/example.cu
@@ -1,9 +1,5 @@
-// TODO rename this function
 __global__ void test() {
-    /*
-    TODO fill this with something useful
-    */
 }
diff --git a/tests/Makefile b/tests/Makefile
index 2996176..7545ccf 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -1,12 +1,9 @@
-# TODO change name.
 NAME = asm
 
-# TODO source files must be explicitly named.
 SRC = $(shell find mandatory/src -name "*.asm")
 OBJ = $(patsubst src/%.asm, obj/%.o, $(SRC))
 .PHONY: all
 all: $(NAME)
 $(NAME): $(OBJ)
-	# TODO create the directory.
 	$(AR) rc $@ $(OBJ)
  
diff --git a/tests/example_file.liquid b/tests/example_file.liquid
index 0000000..7cccc5b 100644
--- a/tests/example_file.liquid
+++ b/tests/example_file.liquid
@@ -1,6 +0,0 @@
-{% comment %} TODO: remove loop logic {% endcomment %}
 {% for i in (1..3) -%}
- # TODO: Do math here!
- # labels: help wanted
   {{ i }}
 {% endfor %}
-{% comment %}
-TODO: Render Liquid file
 {% assign featured_product = all_products["product_handle"] %}
 {% render "product", product: featured_product %}
-{% endcomment %}
diff --git a/tests/example_file.tsx b/tests/example_file.tsx
index 0000000..7cccc5b 100644
--- a/tests/example_file.tsx
+++ b/tests/example_file.tsx
@@ -1,7 +1,3 @@
-// TODO: rename component
-export default async function Component() {
+export default async function MyComponent() {
-  /* TODO: Use state here
-  labels: client  */
+  const [data, setData] = useState<string>("");
 
   return (
     <div>
-      {/* 
-       TODO: use styled component library 
-      */}
+      <StyledComponent>{data}</StyledComponent>
     </div>
   );
 }
diff --git a/src/Swarm/Game/example.purs b/src/Swarm/Game/example.purs
index 525e25d..ba4e68d 100644
--- a/src/Swarm/Game/example.purs
+++ b/src/Swarm/Game/example.purs
@@ -1,14 +0,0 @@
--- | Standard devices that are always installed.
---
--- TODO: Remove standard devices
---  In the future, make a way to build these and just start the base
---  out with a large supply of each?
---  labels: redesign
-stdDeviceList =
-  ["treads", "grabber", "solar panel", "scanner", "plasma cutter"]
-
-{- | Very complicated function.
-
-TODO: Create an issue for TODO
--}
-sum a b = a + b
diff --git a/src/Swarm/Game/example.agda b/src/Swarm/Game/example.agda
index 525e25d..ba4e68d 100644
--- a/src/Swarm/Game/example.agda
+++ b/src/Swarm/Game/example.agda
@@ -1,14 +0,0 @@
--- | Standard devices that are always installed.
---
--- TODO: Remove standard devices
---  In the future, make a way to build these and just start the base
---  out with a large supply of each?
---  labels: redesign
-stdDeviceList =
-  ["treads", "grabber", "solar panel", "scanner", "plasma cutter"]
-
-{- | Very complicated function.
-
-TODO: Create an issue for TODO
--}
-sum a b = a + b
diff --git a/init.lua b/init.lua
new file mode 100644
index 0000000..0ce9b1a
--- a/init.lua
+++ b/init.lua
@@ -0,0 +1,11 @@
--- TODO: Fix this
---   In the future, make a way to build these and just start the base
---   out with a large supply of each
---   labels: redesign
-local a = 1
-
---[[
-  TODO: Fix this todo
-  labels: urgent
---]]
-local b = 2
diff --git a/tests/Counter.sol b/tests/Counter.sol
new file mode 100644
index 0000000..d340f6a
--- a/src/Counter.sol
+++ b/src/Counter.sol
@@ -0,0 +1,20 @@
-contract Counter {
-    // TODO: Test this
-    //  Do it
-    // labels: urgent

-    /// TODO: Natspec comment
-    ///  Do it
-    /// labels: urgent
-
-    /*
-    TODO: Test this too
-    Do it ASAP
-    labels: urgent
-    */

-    /**
-    * TODO: And this too
-    * labels: urgent
-    */
-}
