diff --git a/tests/ExampleFile.java b/tests/ExampleFile.java
index d340f6a..29b54da 100644
--- a/tests/ExampleFile.java
+++ b/tests/ExampleFile.java
@@ -1,13 +1,5 @@
 package com.mydomain.myapp;

 public class JavaTests {
-    // TODO: Expand this class
-    //  Do it
-    // labels: urgent

-    /*
-    TODO: Definitely expand this class
-    Do it ASAP
-    labels: urgent
-    */
 }
\ No newline at end of file
diff --git a/tests/example-file.php b/tests/example-file.php
deleted file mode 100644
index 063bb80..0000000
--- a/tests/example-file.php
+++ /dev/null
@@ -1,23 +0,0 @@
-<!DOCTYPE html>
-<html>
-<body>
-<!-- TODO: Check HTML comments work -->
-<?php
-    echo 'Testing 123';
-    // TODO: Expand this page
-    //  We aren't doing anything here yet
-
-    echo 'Testing 456';
-    # TODO: Definitely expand this
-    #  There needs to be some point to this file
-
-    echo 'Testing 789'
-    /*
-     * TODO: Echo a greeting instead
-     * This should work with or without the asterisk on each line
-     * labels: help wanted
-     */
-?>
-
-</body>
-</html>
\ No newline at end of file
diff --git a/tests/example_file.py b/tests/example_file.py
index 525e25d..ba4e68d 100644
--- a/tests/example_file.py
+++ b/tests/example_file.py
@@ -1,23 +1,2 @@
 def hello_world():
-    # TODO: Come up with a more imaginative greeting
     print('Hello world')
-
-    # TODO: Do more stuff
-    #  This function should probably do something more interesting
-    #  labels: help wanted
-
-def my_method(arg):
-    """
-    TODO: Second multiline comment
-    also need to be turned into task, and hopefully 
-    kept together as one.
-    """
-    pass
-
-def MyClass:
-    '''
-    TODO: Third multiline comment
-    also need to be turned into task, and hopefully 
-    kept together as one.
-    '''
-    pass
diff --git a/tests/example_file.cr b/tests/example_file.cr
index e6da2ec..67f14dd 100644
--- a/tests/example_file.cr
+++ b/tests/example_file.cr
@@ -1,14 +1,3 @@
-# TODO: Come up with a more imaginative greeting
 puts "Greetings"
-
-# TODO: Do even more stuff
-# This function should probably do something more interesting
-# labels: help wanted
diff --git a/tests/example_file.rb b/tests/example_file.rb
index e6da2ec..67f14dd 100644
--- a/tests/example_file.rb
+++ b/tests/example_file.rb
@@ -1,14 +1,3 @@
 #!/usr/bin/ruby -w

-# TODO: Come up with a more imaginative greeting
 puts "Greetings"
-
-=begin
-TODO: Do more stuff
-This function should probably do something more interesting
-labels: help wanted
-=end
-
-# TODO: Do even more stuff
-# This function should probably do something more interesting
-# labels: help wanted
diff --git a/tests/example_file.yaml b/tests/example_file.yaml
index 6397789..494d42f 100644
--- a/tests/example_file.yaml
+++ b/tests/example_file.yaml
@@ -1,7 +1,3 @@
 name: "TODO to Issue"
-# TODO: Write a more interesting description
 description: "Converts IDE TODO comments to GitHub issues"
 author: "Alastair Mooney"
-# TODO: Define inputs
-#  Need to do this before the action is released
-#  labels: urgent
diff --git a/tests/example_file.prog.abap b/tests/example_file.prog.abap
index 6397789..494d42f 100644
--- /dev/null
+++ b/tests/example_file.prog.abap
@@ -1,5 +0,0 @@
-REPORT ztest_todo_2.
-
-DATA moo TYPE i VALUE 2.
-WRITE moo. " TODO This is an end-of-line to-do
-moo = 4. * TODO This is another end-of-line to-do
diff --git a/tests/example_file.sql b/src/tests/example_file.sql
index 6397789..494d42f 100644
--- /dev/null
+++ b/src/tests/example_file.sql
@@ -1,2 +0,0 @@
--- TODO Select all:
-SELECT * FROM Products;
diff --git a/tests/example_file.tex b/src/tests/example_file.tex
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.tex
@@ -0,0 +1,2 @@
-% TODO Add in preamble details
-\begin{document}
-  \begin{comment}
-     TODO This document needs content
-     label: urgent
-  \end{comment}
-\end{document}
diff --git a/tests/example_file.jl b/src/tests/example_file.jl
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.jl
@@ -0,0 +1,2 @@
- # TODO: Hopefully this comment turns into an issue
- print("Hello World")
- #= TODO: Multiline comments
-   also need to be turned into task, and hopefully 
-   kept together as one.
- =#
diff --git a/tests/example_file.ahk b/src/tests/example_file.ahk
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.ahk
@@ -0,0 +1,2 @@
- ; TODO: Find a better way to manage hotkeys
- ; Maybe just switch to vim??
- #h::
- RegRead, HiddenFiles_Status, HKEY_CURRENT_USER, Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced, Hidden 
diff --git a/tests/example_file.hbs b/src/tests/example_file.hbs
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.hbs
@@ -0,0 +1,2 @@
- <!-- TODO: Hopefully this comment turns into a todo issue -->
- {{! 
-   TODO: Make a handlebar templtate
-    This is really just a test, but hopefully this works~!
- }}
diff --git a/tests/example_file.org b/src/tests/example_file.org
new file mode 100644
index 0000000..7cccc5b
--- /dev/null
+++ b/src/tests/example_file.org
@@ -0,0 +1,2 @@
- # TODO: Hopefully this comment turns into a todo issue
- #+begin_src python
- print("Hello World")
- #+end_src
- #+begin_comment
-  TODO: Multiline comments
-   also need to be turned into todos, and hopefully 
-   kept together as one todo
- #+end_comment
diff --git a/tests/example_file.scss b/tests/example_file.scss
index 525e25d..ba4e68d 100644
--- a/tests/example_file.scss
+++ b/tests/example_file.scss
@@ -1,11 +0,0 @@
-// TODO: Hopefully this comment turns into a todo issue
-.my-class {
-  font-weight: bold;
-  .my-second-class {
-    /* TODO: Multiline comments
-     * also need to be turned into todos, and hopefully 
-     * kept together as one todo
-     */  
-    text-align: center;
-  }
-}
diff --git a/tests/example_file.twig b/src/tests/example_file.twig
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.twig
+++ b/src/tests/example_file.twig
@@ -1,6 +0,0 @@
- {# TODO: Hopefully this comment turns into a todo issue #}
- {#
-   TODO:Multiline comments
-   also need to be turned into todos, and hopefully
-   kept together as one todo
- #}
diff --git a/tests/example_file.md b/src/tests/example_file.md
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.md
+++ b/src/tests/example_file.md
@@ -1,6 +0,0 @@
- {/* TODO: Hopefully this comment turns into a todo issue */}
- {/*
-   TODO:Multiline comments
-   also need to be turned into todos, and hopefully
-   kept together as one todo
- */}
diff --git a/tests/example_file.mdx b/src/tests/example_file.mdx
index 525e25d..ba4e68d 100644
--- a/src/tests/example_file.mdx
+++ b/src/tests/example_file.mdx
@@ -1,6 +0,0 @@
- {/* TODO: Hopefully this comment turns into a todo issue */}
- {/*
-   TODO:Multiline comments
-   also need to be turned into todos, and hopefully
-   kept together as one todo
- */}
